### ISSUE TITLE
How to use dense stereo initialization on 6 views?

### ISSUE THREAD

**Issue by Penguin-jpg** [jiaw-z/CoR-GS#27]
I tried to modify the colmap_llff.py file to use it on 6 views, but when I ran it, the below errors occurred.
![image](https://github.com/user-attachments/assets/5010d88c-25f4-42f3-bbd2-ec65fe69cd8b)

I only change the dataset path and the n_views argument. Is there anything I also need to change?

**Comment by jiaw-z** [comment]
Hello, I ran the code and only changed n_views from 3 to 6 as follows, and it works fine. Can you provide more log information?
<img width="1075" alt="image" src="https://github.com/user-attachments/assets/8f0b1858-7ac2-4a2c-aedb-e56f0d721904">

**Comment by Penguin-jpg** [comment]
Thanks for the reply! Here is the complete log:
![image](https://github.com/user-attachments/assets/de621f60-d09d-4ec5-8b0f-6fcb5324c449)
![image](https://github.com/user-attachments/assets/f9d5f35a-6492-45dc-aca6-917c20c89604)
![image](https://github.com/user-attachments/assets/05a63a02-6f89-4a4e-9210-631f1aa2f36e)
![image](https://github.com/user-attachments/assets/758ec695-a7d9-4f14-9472-64a6a263cafa)
![image](https://github.com/user-attachments/assets/8ecd5abc-4a35-4e72-830c-dbd8de222f6d)
![image](https://github.com/user-attachments/assets/263c7265-0c3f-48b7-a602-07d8e81bbeb0)

**Comment by jiaw-z** [comment]
Hello, does the code work fine with 3 input views?
It seems that there are not enough matches during feature matching, but it would be strange if 3-view can find enough matches.
<img width="761" alt="image" src="https://github.com/user-attachments/assets/6356aa60-1531-4e98-8d74-78e10a05c867">

**Comment by Penguin-jpg** [comment]
Yes, 3 input views work fine. I use the docker provided by FSGS. Is the problem related to this?

**Comment by Penguin-jpg** [comment]
Hi, I found that it was because of the environment. Now it is working well, thank you.

**Comment by shaonian123daly** [comment]
Hi, how did you solve it? I also encountered this problem.

**Comment by Penguin-jpg** [comment]
I just reinstalled the environment iirc. However, I encountered this problem again yesterday, and reinstalling didn't work this time.

**Comment by shaonian123daly** [comment]
Sorry to hear that. I'll let you know if I fix this.
